src/backend/oracle-data-api/src/main/resources/tco-openapi-spec.yaml

```diff
@@ -269,6 +269,30 @@ paths:
           $ref: '#/components/responses/Unauthorized'
       tags:
         - JJDispute
+  /v1/deleteDispute:
+    delete:
+      parameters:
+        - name: disputeId
+          in: query
+          description: ''
+          schema:
+            type: integer
+            format: int64
+        - name: ticketNumber
+          in: query
+          schema:
+            type: string
+      responses:
```

Comment:
The API has a different set of return codes.

```
	@Operation(summary = "Deletes a particular JJDispute record.")
	@ApiResponses({
		@ApiResponse(responseCode = "200", description = "Ok. JJ Dispute record deleted."),
		@ApiResponse(responseCode = "400", description = "Bad Request."),
		@ApiResponse(responseCode = "404", description = "JJ Dispute record not found. Delete failed."),
		@ApiResponse(responseCode = "500", description = "Internal Server Error. Delete failed.")
	})
	@DeleteMapping("/dispute")
```

Reply:
Those responses are the ones returned from the ORDS endpoint. The Oracle Data API has different set of API responses because it checks the parameters etc. So those responses defined in `tco-openapi-spec.yaml` are accurate. ORDS only returns 200 and 401.

Reply:
ok